irohad/model/execution/impl/command_executor.cpp

```diff
@@ -48,40 +52,47 @@ bool AddAssetQuantityExecutor::execute(const Command &command,
 
   auto asset = queries.getAsset(add_asset_quantity.asset_id);
   if (not asset.has_value()) {
-    // No such asset
+    log_->info("[AddAssetQuantity] asset {} is absent",
```

Comment:
It would probably be less error-prone to initialize logger with new name in derived classes, than to copy command name every time.